mmv1/products/vertexai/Models.yaml

```diff
@@ -0,0 +1,955 @@
+                                default_value: 1000
+                              - !ruby/object:Api::Type::Integer
+                                name: 'leafNodesToSearchPercent'
+                                description: |-
+                                  The default percentage of leaf nodes that any query may be searched. Must be in
+                                  range 1-100, inclusive. The default value is 10 (means 10%) if not set.
+                                default_value: 10
+                          - !ruby/object:Api::Type::NestedObject
+                            name: 'bruteForceConfig'
+                            allow_empty_object: true
+                            send_empty_value: true
+                            properties: []
+                            exactly_one_of:
+                              - treeAhConfig
+                              - bruteForceConfig
+                            description: |-
+                              Configuration options for using brute force search, which simply implements the
+                              standard linear search in the database for each query.
+              - !ruby/object:Api::Type::NestedObject
+                name: 'presets'
+                exactly_one_of:
+                  - presets
+                  - nearestNeighborSearchConfig
+                description: Simplified preset configuration, which automatically sets configuration values based on the desired query speed-precision trade-off and modality.
+                properties:
+                  - !ruby/object:Api::Type::Enum
+                    name: 'modality'
+                    description: The modality of the uploaded model, which automatically configures the distance measurement and feature normalization for the underlying example index and queries
+                    values:
+                      - :IMAGE
+                      - :TEXT
+                      - :TABULAR
+                  - !ruby/object:Api::Type::Enum
+                    name: 'query'
+                    description: Preset option controlling parameters for speed-precision trade-off when querying for examples. If omitted, defaults to PRECISE.
+                    values:
+                      - :PRECISE
+                      - :FAST
+      - !ruby/object:Api::Type::NestedObject
+        name: 'metadata'
+        description: metadata describing the Model's input and output for explanation.
+        properties:
+          - !ruby/object:Api::Type::Map
+            name: 'inputs'
+            key_name: 'name'
+            description: The resource name of the Artifact that was created in MetadataStore when creating the Model.
+            required: true
+            value_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::String
+                  name: 'sha1Sum'
+                  description: |
+                    SHA1 checksum of the file
```

Comment:
I don't see sha1Sum listed in the fields here: https://cloud.google.com/vertex-ai/docs/reference/rest/v1/ExplanationSpec#InputMetadata

I think that could be why `TestAccVertexAIModels_vertexAiModelUploadExplanationSpecExample` fails with:
```
Invalid JSON payload received. Unknown name \"sha1Sum\" at 'model.explanation_spec.metadata.inputs[0].value':...
```